concent_api/command_line_tool/concent_request.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import argparse
+import json
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
What does this line do ?

Reply:
Allows to import modules outside command line tool package that are not in venv e.g. utils.testing_helpers

Reply:
Isn't that possible by using '..' ?

Reply:
Sorry, bad example. I meant api_testing_common.

Reply:
Ok, makes sense. Maybe just add a comment above like this:
```
# TODO: Remove when importing from api_testing_common is no longer needed.
```

Reply:
Done.